data/severity.go

```diff
@@ -0,0 +1,6 @@
+package data
+
+// Severities consists of severity levels
+var Severities = map[string][]string{
+	"level": {"EMERGE", "ALERT", "CRIT", "ERROR", "WARN", "NOTICE", "INFO", "DEBUG"},
```

Comment:
Is it not `EMERG`?

Reply:
Also doc comment can be more descriptive.

Reply:
Ah sorry, `EMERG` is right. And, If we will provide both short and long versions, `ERROR` should be `ERR` in here.